bitnami/nginx-ingress-controller/values.yaml

```diff
@@ -558,6 +558,18 @@ defaultBackend:
   extraArgs: {}
   containerPort: 8080
 
+  ## NGINX backend default server block configuration
+  ## Should be compliant with: https://kubernetes.github.io/ingress-nginx/user-guide/default-backend/
+  ##
+  serverBlockConfig: |-
+    location /healthz {
+      return 200;
+    }
+
+    location / {
+      return 404;
+    }
+
```

Comment:
I think I would leave this object empty in this file and add this default configuration to the configmap instead. Then, if this object is set, we can replace the default configuration with the one supplied by the user.

Reply:
I think it makes more sense for this to be in `values.yaml`:
- If a chart's user want's to know what is the default value of this he will go looking into `values.yaml` first. Moving it elsewhere will decrease the charts "readability". 
- Currently an empty value on `defaultBackend.serverBlockConfig` means that the `ConfigMap` is not created and no mount occurs in the default backend. This might be used by users that are using a custom default backend and do not wish to have an unused mount on their custom image. The proposed change will modify this, the user will no longer be able to control if the mount occurs or not.

If you disagree i will apply your change as you have more context and knowledge on this.